server/plugin/taskLinkPreview.go

```diff
@@ -106,3 +106,52 @@ func (p *Plugin) PostPullRequestPreview(linkData []string, link, userID, channel
 
 	return post, ""
 }
+
+func (p *Plugin) PostBuildDetailsPreview(linkData []string, link, userID, channelID string) (*model.Post, string) {
+	organization := linkData[3]
+	project := linkData[4]
+	buildID := strings.Split(linkData[6], "&")[0][16:]
```

Comment:
Are we sure that we won't get errors like "Array index out of bounds" or similar errors here?

Reply:
yes as we would be reaching here only when we would be having a valid link which satisfies all the conditions, we are checking that in a separate func with this regex `BuildDetailsLinkRegex`